chef/cookbooks/provisioner/recipes/update_nodes.rb

```diff
@@ -138,16 +167,20 @@
           if mnode.macaddress == mac_list[i]
             ipaddress admin_data_net.address
             options [
-     '      if option arch = 00:06 {
-        filename = "discovery/bootia32.efi";
-     } else if option arch = 00:07 {
-        filename = "discovery/bootx64.efi";
-     } else if option arch = 00:09 {
-        filename = "discovery/bootx64.efi";
-     } else {
-        filename = "discovery/pxelinux.0";
-     }',
-                     "next-server #{admin_ip}"
+             'if exists dhcp-parameter-request-list {
```

Comment:
Style/IndentArray: Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.